repo/packages/M/mysql/4/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.7",
```

Comment:
local volumes in 1.7 

https://docs.mesosphere.com/1.7/usage/storage/persistent-volume/